common/resource/resource.go

```diff
@@ -101,7 +99,6 @@ type (
 		GetMetadataManager() persistence.MetadataManager
 		GetClusterMetadataManager() persistence.ClusterMetadataManager
 		GetTaskManager() persistence.TaskManager
-		GetVisibilityManager() visibility.VisibilityManager
```

Comment:
in the long term where do you plan to put the visibility manager? or are you waiting for the DI to do the job?

Reply:
Yes, long term plan: DI should inject it in frontend handler and visibility task executor.